test.sh

```diff
@@ -1,3 +1,3 @@
 cd test
 nosetests --with-coverage --cover-package pyecharts --cover-package test
-mv .coverage ../
```

Comment:
This line hides the error from nosetests and all failing tests are hidden because mv succeeds and returns OK

Reply:
got it.